main_api.py

```diff
@@ -46,7 +40,7 @@
 app = FastAPI()
 app.add_middleware(
     CORSMiddleware,
-    allow_origins=origins,
+    allow_origins=['*'],
```

Comment:
This particular line change replaces the list of specific origins with a wildcard ('*'), allowing all domains to access resources from our application. Was this change intended, or was it made by mistake? @EndeavorPal @waleedkadous

Reply:
I think its intended for mobile (where the origin is not defined). Its not detrimental because of the additional origins check on line 61.

Reply:
Yes, this to handle mobile, also origin validation already handled in line 67
```python
if origin in origins or mobile == "ANSARI":
```